src/plugins/ui_stateful/clipboard.ts

```diff
@@ -135,8 +155,9 @@ export class ClipboardPlugin extends UIPlugin {
             bottom: multipleRowsInSelection ? zone.top : zone.top - 1,
             top: multipleRowsInSelection ? zone.top : zone.top - 1,
           };
-          const state = this.getClipboardStateForCopyCells([copyTarget], "COPY");
-          state.paste([zone], {
+          this.originSheetId = this.getters.getActiveSheetId();
+          this.copiedData = this.copy("COPY", [copyTarget]);
+          this.paste([zone], {
             pasteOption: undefined,
             shouldPasteCF: true,
```

Comment:
`shouldPasteCF` is still "une entorce" to a generic clipboard (which is not supposed to know exactly what is copy/pasted). If it's not trivial to fix, let's keep it that way for now and merge.